hack/release/release.sh

```diff
@@ -8,7 +8,7 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 source "${SCRIPT_DIR}/common.sh"
 
 config
-release "$HEAD_HASH" #release a snapshot version
+snapshot "$HEAD_HASH" #release a snapshot version
```

Comment:
Does it make sense for the release trigger to both create a snapshot and create an official release? I think we just need one. We should already have a snapshot from the merge when we tag

Reply:
RIght now the `install-karpenter` action relies upon the snapshot release, even for official releases. We could update it so it would check for both but I honestly think its simpler to keep it as is.

Reply:
> action relies upon the snapshot release, even for official releases

I'm not quite getting what you mean here. Install karpenter is only used for snapshot releases so why do we need to handle this for the official release?

Reply:
Discussed offline, we've separated the release workflow into two separate workflows: release and snapshot.